borg/helpers.py

```diff
@@ -8,6 +8,11 @@
 import os
 import pwd
 import re
+try:
+    from shutil import get_terminal_size
+except ImportError:
+    def get_terminal_size(fallback):
+        return (os.environ.get('COLUMNS', fallback[0]), os.environ.get('LINES', fallback[1]))
```

Comment:
i wished to keep the alternate implementation as simple as possible... now it's a two-liner, the original implementation is like 30 lines and depends on *another* method in `os`... so a little tangled up and harder to port... so i'll just add the fallback and the namedtuple.